LK_proposals/LK Proposal for Collaboration.md

```diff
@@ -0,0 +1,116 @@
+
+<ins>Procedure
+
+1. 2 seperate preregistrations = simulation protocol Morris et al. (2019)
+    
+    —> remaining RDoF/QRPs in operationalization, hypothesis generation, method selection and interpretation
+    
+
+*Open question*: Should we Red Team on our opposites preregistration? Would mitigate Bias and would differ from AC, as one would not have to follow critique if it feels unwarrented
+
+2. 2 seperate implementations and interpretations
+
+<br/>
+
+**Second round: joint study**
+
+Starting point: Results of first round. Either: 
+
+1. contradicting findings
+2. subjective interpretation of ambiguous results
+3. clear reasoning for why results were not as hypothesized combined with strong opinion on what should be happening in the joint study
+
+In any case: clear divide in opinions, ideal grounds for adversary!
+
+<ins>Along the way:
+
+Constant diary entry after each following part, judging the collaboration quality in a guided fashion, oriented on the diary by Shah & Leeder (2016).
+
+Module: “Propose, Critique & Merge”: Oriented on a guideline including techniques such as in Kardos & Dexter (2017): Key assumption check, What if & counter arguments, Quality of information.
+
+In emergencies as in fundamentally unresolvable disagreements, a third neutral arbiter (Aaron Peikert) should be consulted to try to resolve them.
+
+<br/>
+
+<ins>First part: Joint preregistration: Steps 1-8
+
+Agree on a joint preregistration (including when which hyp. are accepted/ rejected, what are valid interpretations, which exact methods are to be chosen) (Mellers et al., 2001; Clark & Tetlock, 2021; Cowan et al., 2020)
+
+Based on “individual proposal” i.e. revised initial preregistration of first study integrating its results
+
+“Propose, Critique & Merge”
+
+Verbal description of Simulation = simulation protocol Morris et al. (2019), as done in Pawel et al. (2023)
+
+Should be as detailed and comprehensive as possible. Agreed upon steps as well as disagreements should be documented and resolved utilizing Github extensively.
+
+<br/>
+
+<ins>Second part: Coding & execution: Step 9
+
+Individual proposals (code scripts)
+
+“Propose, Critique & Merge”
+
+Debugging should be done jointly.
+
+In the end, all results should be present as defined in preregistration. 
+
+Any changes from the preregistration should be argued and agreed upon, ideally on an empirical basis.
+
+<br/>
+
+<ins>Third part: Interpreting results, Discussion & implications: Step 10
+
+Individual proposals
+
+“Propose, Critique & Merge”
+
+Map disagreements
+
+Ideally, a joint Third part is generated. Alternatively, There have to be seperate interpretations and/ or discussion & implication parts.
+
+<br/>
+
+<ins>Fourth part: Reflecting on the AC
+
+Individual proposals, based on the diaries
```

Comment:
in how far will the diaries play a role here?

Reply:
Good point. This could vary greatly depending on the amount of data the diaries will generate and the focus we want to set in our study.

Reply:
I think the comprehensiveness of the diary use and result analyses should be an important point of discussion.